client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,10 +45,44 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
+				<p>
+					<FormLabel>
+						<FormRadio
+							name="auth_method"
+							value="app-based"
+							defaultChecked={ true }
+							onChange={ setAuth }
+						/>
+						<Gridicon icon="phone" />
+						<span className="security-2fa-initial-setup__item-title">
+							{ this.props.translate( 'Set up using an app' ) }
+						</span>
+						<span className="security-2fa-initial-setup__description">
```

Comment:
FWIW there's a `FormSettingExplanation` component in case that can do the job for you. Just note that it usually should be placed outside of the label, because it produces a block `<p>` tag. IMHO that's fine - I wouldn't expect the description to be part of the label.